docs/cloud-native-security/cloud-workload-protection.asciidoc

```diff
@@ -0,0 +1,19 @@
+[[cloud-workload-protection]]
+= Cloud workload protection
+
+Cloud workload protection helps you monitor and protect your Linux hosts and Kubernetes runtimes. It uses the <<install-endpoint,{elastic-defend}>> integration to capture cloud workload telemetry containing process, file, and network activity. In Kubernetes environments, this telemetry is further enriched with metadata that enables you to isolate events in your cloud topography.
+
+This telemetry also enables the automated identification of cloud threats with out-of-the-box detection rules and machine learning models. Alerts based on these detections can reduce your mean time to identify and remediate threats.
```

Comment:
I'm on the fence about `mean time` here, esp. for our non-native English speakers. Could also be misconstrued for `meantime`. Can we omit `mean` here? If we have to use it, I'd replace with `average`. 
```suggestion
This telemetry also enables the automated identification of cloud threats with out-of-the-box detection rules and machine learning models. Alerts based on these detections can reduce the time to identify and remediate threats.
```